packages/ui/src/components/molecules/Rating/styles.scss

```
@@ -4,22 +4,24 @@
   // --------------------------------------------------------
 
   // Default properties
-  --fs-rating-gap                     : var(--fs-spacing-0);
-  --fs-rating-color                   : var(--fs-color-text);
+  --fs-rating-gap: var(--fs-spacing-0);
+  --fs-rating-color: var(--fs-color-main-2);
+  --fs-rating-color-empty: var(--fs-color-neutral-4);
 
   // Icon
-  --fs-rating-icon-width              : var(--fs-spacing-3);
-  --fs-rating-icon-height             : var(--fs-rating-icon-width);
+  --fs-rating-icon-width: var(--fs-spacing-3);
+  --fs-rating-icon-height: var(--fs-rating-icon-width);
 
   // Actionable
-  --fs-rating-actionable-gap          : 0;
-  --fs-rating-actionable-icon-width   : var(--fs-rating-icon-width);
-  --fs-rating-actionable-icon-height  : var(--fs-rating-actionable-icon-width);
+  --fs-rating-actionable-gap: 0;
+  --fs-rating-actionable-icon-width: var(--fs-rating-icon-width);
+  --fs-rating-actionable-icon-height: var(--fs-rating-actionable-icon-width);
+  --fs-rating-actionable-color: var(--fs-rating-color-empty);
+  --fs-rating-actionable-color-selected: var(--fs-rating-color);
```

```suggestion
  --fs-rating-actionable-icon-color: var(--fs-rating-color-empty);
  --fs-rating-actionable-icon-color-selected: var(--fs-rating-color);
```

Those 2 new tokens is only related to the icon, right?

---

Done ✅